api/workAllocation2/util.ts

```diff
@@ -197,8 +197,8 @@ export function getActionsByPermissions(view, permissions: TaskPermission[]): Ac
   return actionList;
 }
 
-export function applySearchFilter(person: Person, domain: PersonDomain, searchTerm: any): boolean {
-  if (domain === PersonDomain.BOTH) {
+export function applySearchFilter(person: Person, domain: string, searchTerm: any): boolean {
+  if (domain === 'ALL') {
```

Comment:
Why is this being changed to a string as opposed to keeping the enumerable value?

Reply:
the old PersonDomain is with
JUDICIAL = 1,
CASEWORKER = 2,
BOTH = 3,

We can not pass 1, 2, 3 from the container and it's not meaningful, we can still use the string enum, but giving that there is no difference on using string enum, so a common const string is almost the same.

Reply:
I don't think we should be using 'ALL', we should be using the enumerated string.

Reply:
it's now enum, could you double check